osi_hostvehicledata.proto

```diff
@@ -2,31 +2,273 @@ syntax = "proto2";
+
+        // Position of the clutch pedal.
+        // Range: 0-1 (Unpressed - fully pressed)
+        //
+        optional double pedal_position_clutch = 2;
+
+        // The actual gear of the transmission.
+        // For example, a gear lever can be on "D" and the transmission on "4", but not the
+        // other way around.
+        //
+        // The sign of this field is linked to the gear's mode as following:
+        // - zero: neutral position
+        // - positive: driving forward mode
+        // - negative: reverse mode (generally -1, but few vehicles have more than 1
+        //   reverse mode gears)
+        //
+        optional int32 gear_transmission = 3;
+
+        // Information about the motor(s).
+        //
+        repeated Motor motor = 4;
+
+        //
+        // \brief A description of the motor states.
+        //
+        message Motor
+        {
+            // The type of the motor.
+            //
+            optional Type type = 1;
+
+            // Revolutions per minute of the motor.
+            //
+            // Unit: 1/min
+            //
+            optional double rpm = 2;
+
+            // Torque of the motor.
+            //
+            // Unit: N*m
+            //
+            optional double torque = 3;
+
+            // Definition which type of motor is used.
+            //
+            enum Type
+            {
+                // The motor type is unknown.
+                //
+                TYPE_UNKNOWN = 0;
+
+                // It is another motor type.
+                //
+                TYPE_OTHER = 1;
+
+                // A motor working after the principle of Nicolaus Otto.
+                //
+                TYPE_OTTO = 2;
+
+                // A motor working after the principle of Rudolf Diesel.
+                //
+                TYPE_DIESEL = 3;
+
+                // A motor working electric.
+                //
+                TYPE_ELECTRIC = 4;
+            }
+        }
+    }
+
+    //
+    // \brief The focus here is on the description of the brake system. 
+    //
+    message VehicleBrakeSystem
+    {
+        // Position of the brake pedal.
+        // Range: 0-1 (Unpressed - fully pressed)
+        //
+        optional double pedal_position_brake = 1;
+    }
+
+    //
+    // \brief The focus here is on the description of the steering train. 
+    //
+    message VehicleSteering
+    {
+        // Description of the steering wheel.
+        //
+        optional VehicleSteeringWheel vehicle_steering_wheel = 1;
+    }
+
+    //
+    // \brief The focus here is on the description of the wheels.
+    //
+    message VehicleWheels
+    {
+        // Description of each wheel.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated WheelData wheel_data = 1;
+
+        //
+        // \brief The focus here is on the description of a wheel.
+        //
+        message WheelData
```

Comment:
CCB Output 13.10.2021:
1. @pmai Please update the comment to indicate that this only relevant to the internal wheel data of the  vehicle.

Reply:
I already did the description improvements.